lib/archethic_web/explorer/live/transaction_details_live.ex

```diff
@@ -332,7 +332,7 @@ defmodule ArchethicWeb.Explorer.TransactionDetailsLive do
   end
 
   def print_state(%UnspentOutput{encoded_payload: encoded_state}) do
-    encoded_state |> State.deserialize() |> elem(0) |> Jason.encode!(pretty: true)
+    encoded_state |> State.deserialize() |> elem(0) |> inspect(pretty: true)
```

Comment:
I don't know if this is the right, if you take the graphql API, the request will fail
```graphql
query {
   transaction(address: "00005B002866E38C9F11303EAE810C422E9CF61E8FA794EC966219D5C4D82B32BEA5") {
		validationStamp{
      ledgerOperations{
        unspentOutputs {
          state
        }
      }
    }
  }
}
```

If you use `inspect` the output is not anymore a valid JSON